frontend/packages/core-api-client/src/lib/client/Recipient.ts

```diff
@@ -0,0 +1,27 @@
+import {FormLookup, RecordLookup, Response} from "../types";
+import {
+  Recipient,
+  RecipientDefinition,
+  RecipientList,
+} from "../types/client/Recipient";
+import { RecordClient } from "./Record";
+
+export class RecipientClient {
+  recordClient: RecordClient;
+
+  constructor(recordClient: RecordClient) {
+    this.recordClient = recordClient;
+  }
+
+  create = (recipient: RecipientDefinition): Promise<Recipient> => {
+    return this.recordClient.create(recipient);
+  };
```

Comment:
Based on my other comment this would become:

```
  create = async (record: RecipientDefinition): Promise<Recipient> => {
    const response = this.recordClient.create({ object: record });
    if (response.success)
      return (response as SuccessResponse<ExampleDefinition, Example>).response;
    throw new APIError((response as ErrorResponse<ExampleDefinition, Example>).error); // TODO define a custom error
  };
```